src/api/payment-requests/payment-requests.md

```diff
@@ -103,6 +104,19 @@ version (documented on this page) and the "legacy" version (documented at
 ★  For payment options which specify an address, there's a requirement to make a transaction on an external ledger.
 Once you have made that payment, you can use the transaction id to [Pay a Payment Request][] using the legacy payment API.
 
+### Merchant Condition
+Some asset types may require conditional operator approval. Without conditions enabled, these asset types may not be acceptable for payment.
```

Comment:
Maybe something along the lines of - Some payment options may require conditional operator approval. Without the merchant condition approved, the payment option will not be an option.
+ the conditions enabled flag.